codec-http/src/test/java/io/netty/handler/codec/http/multipart/HttpPostRequestEncoderTest.java

```diff
@@ -110,6 +113,38 @@ public void testMultiFileUploadInMixedMode() throws Exception {
         assertEquals(expected, content);
     }
 
+    @Test
+    public void testHttpPostRequestEncoderSlicedBuffer() throws Exception {
+        DefaultFullHttpRequest request = new DefaultFullHttpRequest(HttpVersion.HTTP_1_1,
+                HttpMethod.POST, "http://localhost");
+
+        HttpPostRequestEncoder encoder = new HttpPostRequestEncoder(request, true);
+        // add Form attribute
+        encoder.addBodyAttribute("getform", "POST");
+        encoder.addBodyAttribute("info", "first value");
+        encoder.addBodyAttribute("secondinfo", "secondvalue ���&");
```

Comment:
Yes ;-) It was directly taken from the example to ensure the result was the same...
Now it is fixed.